packages/create-react-app/index.js

```diff
@@ -69,6 +69,7 @@ var program = require('commander')
   })
   .option('--verbose', 'print additional logs')
   .option('--scripts-version <alternative-package>', 'use a non-standard version of react-scripts')
+  .option('--template <path-to-variant>', 'to select a template variant')
```

Comment:
Please change to `--template <path-to-template> 'use a non-standard application template'`.